src/D2L.Bmx/BmxConfigProvider.cs

```diff
@@ -53,6 +56,26 @@ internal class BmxConfigProvider : IBmxConfigProvider {
 		);
 	}
 
+	public void SaveConfiguration( BmxConfig config ) {
```

Comment:
So I can change it back to not use reflection if anyone has an issue with it

Reply:
Please no reflection...

Reply:
We need to do mapping anyway, e.g. `DefaultDuration` <-> `default_duration`, so the reflection code is technically wrong here.
Reflection isn't trimming or AOT friendly in general, so we want to avoid that especially in bmx unless there's a very compelling reason otherwise.

Reply:
Haha okay. I will just use WriteLine for each one. I didn't know about the trimming worries and just saw when publishing to aot it was working

Reply:
I originally thought to save it as json thinking that'd be easiest but then we'd break existing setups